inst/conf/config-template.yml

```diff
@@ -4,9 +4,13 @@ default:
   spark.sql.shuffle.partitions.local: !expr parallel::detectCores()
   spark.env.SPARK_LOCAL_IP.local: 127.0.0.1
 
+  # default embedded packages to load
+  sparklyr.jars.default:
+    - !expr system.file(file.path("java", "spark-csv_2.11-1.3.0.jar"), package = "sparklyr")
+    - !expr system.file(file.path("java", "commons-csv-1.1.jar"), package = "sparklyr")
```

Comment:
You may also want to include uniVocity-parsers jar file. spark-csv supports an option to chose parser library. Some features only work with univocity. Also there is slight difference in parsing behavior.